NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache Daffodil Schema Template
+Copyright 2021 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
This NOTICE doesn't mention any specific software - consider removing this file or mentioning the giter8 software.

Reply:
The "software" in this case is all this source code. I believe this is the standard boilerplate that goes in Apache notice files.

Reply:
FYI, here's there reference: https://www.apache.org/legal/src-headers.html#notice